package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@react-icons/all-files": "^4.1.0",
+    "@reduxjs/toolkit": "^1.8.5",
```

Comment:
`redux toolkit` is present in `package.json` 🔥

Reply:
What does it mean ?